explainaboard/metrics/auxiliary/qa_table_text_hybrid_auxiliary.py

```diff
@@ -172,7 +163,9 @@ def ws_tokenize(text):
     return tokens
 
 
-def _answer_to_bags(answer: Union[str, list[str], tuple[str, ...]]):
+def _answer_to_bags(
+    answer: Union[str, list[str], tuple[str, ...]]
```

Comment:
There looks to be mixed use of Union and the or operator.

```suggestion
    answer: str | list[str] | tuple[str, ...]
```